frontend_tests/node_tests/compose.js

```diff
@@ -161,6 +161,19 @@ people.add(bob);
     assert.equal($('#error-msg').html(), 'Please specify a topic');
 }());
 
+(function test_get_invalid_recipient_emails() {
+    var fs_bot = {
+        email: 'fs@example.com',
+        user_id: 124,
+        full_name: 'File System bot',
```

Comment:
Just to have more realistic test data, let's name this feedback_bot with email 'feedback@example.com' and name 'Feedback Bot'.